src/commonTest/resources/dummy-platform-new-model/hybris/bin/modules/api-registry/apiregistryservices/extensioninfo.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I created a new test dummy platform to have the modules in the new model, instead of ext- like we already have.